docs/guides/THREADING.md

```
@@ -470,12 +638,228 @@ async def safe_async_processing():
+# Access from different thread - will raise!
+def worker():
+    import asyncio
+    asyncio.run(node.process(input_data))  # Raises ModelOnexError!
+
+thread = threading.Thread(target=worker)
+thread.start()
+thread.join()
+```
+
+Error output:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)